test/test_VTK.jl

```diff
@@ -1,5 +1,7 @@
 using SHA
 @testset "VTK" begin
+    if VERSION.minor < 6 # temporarily skip tests on this until
```

Comment:
Skipping 0.6 tests here. Seems to work outside the test suite so thats nice at least.

```julia
julia> function foo()
           grid = generate_grid(Triangle, (2, 2))
           vtk = vtk_grid("testfile", grid)
           vtk_save(vtk)
       end
foo (generic function with 1 method)

julia> foo()
1-element Array{String,1}:
 "testfile.vtu"
```